tests/appsec/waf/test_addresses.py

```diff
@@ -11,8 +11,9 @@
     pytestmark = pytest.mark.skip("not relevant")
 
 
-@released(golang="?", dotnet="?", java="?", php="?", python="?", ruby="?")
+@released(golang="?", dotnet="?", java="?", php="0.1.0", python="?", ruby="?")
```

Comment:
if it's not releqsed at all

```suggestion
@released(golang="?", dotnet="?", java="?", php="?", python="?", ruby="?")
```

Reply:
This test shouldn't work on any agent because it's a limitation of the WAF/rules. I thought this merited an explanation, but I can leave it as a comment instead.